charts/validator/values.yaml

```diff
@@ -169,9 +169,6 @@ ghost:
           # - path: /webapi
```

Comment:
It is very unlikely that we would expose `webapi` here.
As an example, it's OK, but the `metrics` below are configured elswhere and keeping them here would confuse folks.